test/test_nn.py

```diff
@@ -5129,6 +5129,20 @@ def smoothl1loss_no_reduce_test():
         fullname='log_softmax_dim3',
         pickle=False,
     ),
+    dict(
+        fullname='Unfold',
+        constructor=lambda: nn.Unfold((2, 2), (1, 1), (0, 0), (1, 1)),
+        input_size=(2, 4, 3, 3),
+        check_gradgrad=False,
+        test_cuda=True,
+    ),
+    dict(
+        fullname='Fold',
+        constructor=lambda: nn.Fold((3, 3), (2, 2), (1, 1), (0, 0), (1, 1)),
```

Comment:
Could you please add another test that tests having integer inputs, instead of tuples, so we can have better test coverage for the API?